ChangeLog

```diff
@@ -4,6 +4,11 @@ Pylint's ChangeLog
 What's New in Pylint 2.0?
 =========================
 
+    * Add a check `chained-comparison` for checking chained comparisons
```

Comment:
Please also add a new entry in `doc/whatsnew/2.0.rst` with some examples on how it's supposed to work.